subprojects/gradle/deeplink-generator/src/main/kotlin/com/avito/deeplink_generator/DeeplinkGeneratorExtension.kt

```diff
@@ -0,0 +1,17 @@
+package com.avito.deeplink_generator
+
+import com.avito.deeplink_generator.internal.parser.DeeplinkParser
+import com.avito.deeplink_generator.model.Deeplink
+
+public open class DeeplinkGeneratorExtension {
+
+    public var publicDeeplinks: Set<Deeplink> = emptySet()
+
+    public var defaultScheme: String = ""
+    public var activityIntentFilterClass: String = ""
+    public var validationCodeFixHint: String = ""
+
```

Comment:
Instead of empty values it would be more conventionally to use Properties.
They can easily distinguish not set values and add default ones if needed.

Reply:
Got it, I'll use Properties